corehq/elastic.py

```diff
@@ -134,9 +134,9 @@ def run_query(url, q):
     return get_es().get(url, data=q)
 
 
-def es_histogram(histo_type, domains=None, startdate=None, enddate=None, tz_diff=None,
-        interval="day", q=None, filters=[]):
-    q = q or {"query": {"match_all":{}}}
+def es_histogram(histo_type, domains=None, startdate=None, enddate=None,
+        interval="day", filters=[]):
```

Comment:
`es_histogram` is only used in one place, and that call doesn't use the `tz_diff` or `q` parameters, so I removed them.  It'd be nice to switch this function over, but I'm less familiar with that and just decided to punt for now.